### ISSUE TITLE
Background scraping only

### ISSUE THREAD

**Issue by jaimsitecom** [Tikipeter/repository.tikipeter#128]
Hi. Hoping you’d put back the scraping notification back to foreground. I noticed when it went to the background it became slower and pauses more, which anybody can mistake as hanging or it selection didnt go thru. 

Another issue is ‘changes log’ dont appear anymore after update and dont appear at tools also when selected.

**Comment by Tikipeter** [comment]
Hi @jaimsitecom 

Change scraping notification back to normal through Settings->Playback->Playback->Autoplay First Relevant Result and change "Use Minimal Notifications" to OFF. Minimal Notifications is available for Autoplay only.

I am still able to open the log through Tools, and the log is still showing when an update occurs. Are you getting an error when it fails?

**Comment by Tinuz97** [comment]
Just a idea, is it a idea to contact A4K (open scrapers) to use the 'old' TikiScrapers as a base,and develop that further? (hope that is correct English).

That was amazingly fast ,i do miss TikiScrapers :/

**Comment by Tikipeter** [comment]
I know you miss it. Me too. :(

I think the new extra slowness at initializing has to do with a new cfscrape update. Likely Tikiscrapers would have had to incorporate that as well, meaning it would have suffered from the same slowness.

If/when my pull request at Openscrapers is accepted, that will make it SLIGHTLY quicker getting the scrape started, but it won't fix the slow start problem entirely.

One way to help yourself after the pull request is accepted is to turn on automatic disabling of failing scrapers in Fen, or at least turn off all the foreign scrapers in Openscrapers. Turn off as many scrapers as you can without affecting results too much, and there will be a speed increase. But only after the pull request is accepted. Until then we're all suffering together. :( :(

**Comment by jaimsitecom** [comment]
Thanks for the quick reply!

As far as the options in playback, i only am fetting these:
[cid:80CB11AB-5D7D-485F-8B2F-3AFB4B7975FD-L0-001]

Do you think an uninstall/install would help rebuild the addon to what options it should have?  I’ve just been doin routine auto install updates whenever.  Or is there a refresh inside the tools/settings?

Thanks for the help!

On Feb 13, 2020, at 10:42 PM, Tikipeter <notifications@github.com> wrote:

﻿

Hi @jaimsitecom<https://github.com/jaimsitecom>

Change scraping notification back to normal through Settings->Playback->Playback->Autoplay First Relevant Result and change "Use Minimal Notifications" to OFF. Minimal Notifications is available for Autoplay only.

I am still able to open the log through Tools, and the log is still showing when an update occurs. Are you getting an error when it fails?

**Comment by Tikipeter** [comment]
I think you may have posted a picture just now, unfortunately I cannot see it. Could you look at your formatting of your message?

**Comment by jaimsitecom** [comment]
> Hi @jaimsitecom
> 
> Change scraping notification back to normal through Settings->Playback->Playback->Autoplay First Relevant Result and change "Use Minimal Notifications" to OFF. Minimal Notifications is available for Autoplay only.
> 
> I am still able to open the log through Tools, and the log is still showing when an update occurs. Are you getting an error when it fails?
> 
> EDIT: I saw your post on Reddit. The slow start to scraping is due to changes to Openscrapers, not Fen. It takes sometimes much longer to initialize now than it did a couple of versions ago.
> 
> I've put back the 'busy' dialog before scraping starts for next update.

Thanks again!  I guess i’ll just have to wait for openscrapers to ‘optimize’ their code to speed up things like before. :)

I love FEN man! Thanks!

**Comment by jaimsitecom** [comment]
> I think you may have posted a picture just now, unfortunately I cannot see it. Could you look at your formatting of your message?

I edited my first response with an image. I forgot that i downgraded to 1.5.96. So i updated again and the option showed up. And it worked! Ty!
